WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModel.kt

```diff
@@ -109,6 +127,19 @@ class PrepublishingHomeViewModel @Inject constructor(
         _uiState.postValue(prepublishingHomeUiStateList)
     }
 
+    private fun onStoryTitleChanged(storyTitle: String) {
+        updateStoryTitleJob?.cancel()
+        updateStoryTitleJob = launch(bgDispatcher) {
+            delay(THROTTLE_DELAY)
```

Comment:
could you explain why this is needed? Are we using this delay elsewhere? thanks

Reply:
Oh, so this was used in the `PrepublishingTagsViewModel`( [link to file](https://github.com/wordpress-mobile/WordPress-Android/blob/db1d3611f9b745fc0532f3990daf7be6bf153a5a/WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingTagsViewModel.kt)) so that there's a delay since every single character change received would cause a save operation so I just used the same approach here. Let me know what you think.

Reply:
That explains it very well, could we add a comment to code explaining this? 🙏

Reply:
Resolved here 6427330